backend/src/server.js

```diff
@@ -2,11 +2,13 @@ const express = require('express');
 const path = require('path');
 const cors = require('cors');
 const morgan = require('morgan');
+
 // const contactRouter = require('./api/contacts/contactRoutes');
 // const userRouter = require('./api/users/userRoutes');
-// const authRouter = require('./api/auth/authRoutes');
-const dotenv = require('dotenv');
-dotenv.config();
+const authRouter = require('./api/auth/auth.routers');
+// const dotenv = require('dotenv');
+require('dotenv').config({ path: path.join(__dirname, './.env') });
```

Comment:
`.env` file should be near `.env.example`

Reply:
ok